packages/slice-machine/src/legacy/lib/builders/SliceBuilder/FieldZones/index.tsx

```diff
@@ -248,11 +248,11 @@ const FieldZones: FC = () => {
           <Box padding={24} gap={12} flexDirection="column">
             <strong>
               This action will permanently remove the repeatable zone from the{" "}
-              {slice.model.name} slice.
+              {slice.model.name} slice {variation.name} variation.
```

Comment:
:question: #ask

Should we only add all this difference if there are more than one variation? It can be strange if you have only one, no?